.github/issue-branch.yml

```diff
@@ -0,0 +1,40 @@
+# see https://github.com/robvanderleek/create-issue-branch
+
+# The default mode is "auto", meaning a new issue branch is created after an issue is assigned.
```

Comment:
Why not use the "auto" mode then?

Reply:
I really want to avoid over-using tools. I don't want to have to be afraid of actions in the GitHub UI starting some ominous background tasks doing _anything_. I want to explicitly start those actions so that when I accidentally assign the wrong issue or if I just want to assign an issue to someone some time _prior_ to actual implementation (which happens not too rarely), no branch is created. The latter point is almost the most important one - assignees might be set in advance for planning/coordination purposes - then I don't need (nor want) a stale branch to be created. The assignee might even change in-between (i.e. before the actual implementation start). Furthermore, if I rename the issue in-between, the old branch is still there, a new one will be created during re-assignment and you have to rename and/or delete the old one which was previously created. And finally, it would also differ from the workflow in GitLab - when we consider clicking the button and issuing `/cib` equivalent (i.e. one minor UI interaction), then nothing changes....some background magic during issue assignment, however, is a leap I'm not too comfortable with.

If my points don't convince you in any way, I can change it to `auto` - frankly, a little begrudgingly, though.

Reply:
Okay, makes sense.